app/upgrade_handler.go

```diff
@@ -414,3 +417,16 @@ func prefixToShentuAddrs(addrs []string) (newAddrs []string, err error) {
 	}
 	return newAddrs, err
 }
+
+func transDescriptionForBank(ctx sdk.Context, app ShentuApp) (err error) {
+	baseKeeper := app.BankKeeper.BaseKeeper
+	metaData := baseKeeper.GetAllDenomMetaData(ctx)
+
+	for _, meta := range metaData {
+		if meta.Description == "The native staking token of the CertiK Chain." {
+			meta.Description = "The native staking token of the Shentu Chain."
+		}
+		baseKeeper.SetDenomMetaData(ctx, meta)
```

Comment:
put this line in the if statement?

Reply:
we can also remove the if directly and assign the value directly

Reply:
Because although it is an array at present, the length is only 1 and will not affect other